6-jre7/Dockerfile

```diff
@@ -34,5 +34,18 @@ RUN set -x \
 	&& rm bin/*.bat \
 	&& rm tomcat.tar.gz*
 
+RUN set -x \
+	&& apt-get update \
+	&& apt-get install -yq gcc make openssl libssl-dev libapr1 libapr1-dev openjdk-7-jdk="$JAVA_DEBIAN_VERSION" \
+	&& tar zxf /usr/local/tomcat/bin/tomcat-native.tar.gz -C /tmp \
+	&& cd /tmp/tomcat-native*-src/jni/native/ \
+	&& ./configure --with-apr=/usr/bin/apr-1-config --with-java-home=/usr/lib/jvm/java-7-openjdk-amd64/ --with-ssl=yes --libdir=/usr/lib/jni \
```

Comment:
The java-home should be cleaner once the https://github.com/docker-library/java/pull/61 is pushed to the Docker Hub.